### PR TITLE
Fix issue #4

### PR DESCRIPTION
**Pull request by mabdi**
I extract the old and the new nodes when the mutated sourcecode is generating in `MutantOperator>>#modifiedSourceFor:with:number:`. 

Then, I set these values to `mutatedNode` and `nodeToMutate` variables in `MethodMutation`. 

So, there is no need to call `lookupForNodes` anymore.

### CODE REVIEW

File: src/MuTalk-Model/MethodMutation.class.st [hunk 1]
```diff
@@ -9,7 +9,7 @@ Class {
 		'mutatedNode',
 		'nodeToMutate'
 	],
-	#category : 'MuTalk-Model'
+	#category : #'MuTalk-Model'
```
Comment (pavel-krivanek) [inline src/MuTalk-Model/MethodMutation.class.st]:
```suggestion
	#category : 'MuTalk-Model'
```

File: src/MuTalk-Model/MutantOperator.class.st [hunk 2]
```diff
@@ -1,7 +1,11 @@
 Class {
 	#name : #MutantOperator,
 	#superclass : #Object,
-	#category : 'MuTalk-Model-Operators'
+	#instVars : [
+		'newNode',
+		'oldNode'
+	],
+	#category : #'MuTalk-Model-Operators'
```
Comment (pavel-krivanek) [inline src/MuTalk-Model/MutantOperator.class.st]:
```suggestion
	#category : 'MuTalk-Model-Operators'
```

File: src/MuTalk-TestResources/ClassForTestingCoverage.class.st [hunk 3]
```diff
@@ -1,14 +1,22 @@
 Class {
 	#name : #ClassForTestingCoverage,
 	#superclass : #Object,
-	#category : 'MuTalk-TestResources'
+	#category : #'MuTalk-TestResources'
```
Comment (pavel-krivanek) [inline src/MuTalk-TestResources/ClassForTestingCoverage.class.st]:
```suggestion
	#category : 'MuTalk-TestResources'
```

File: src/MuTalk-Tests/MethodMutationTest.class.st [hunk 4]
```diff
@@ -1,7 +1,7 @@
 Class {
 	#name : #MethodMutationTest,
 	#superclass : #TestCase,
-	#category : 'MuTalk-Tests'
+	#category : #'MuTalk-Tests'
```
Comment (pavel-krivanek) [inline src/MuTalk-Tests/MethodMutationTest.class.st]:
```suggestion
	#category : 'MuTalk-Tests'
```

**Comment by mabdi** [comment]
Hello Pavel,

I'm using:

```
Pharo 8.0.0
Build information: Pharo-8.0.0+build.1124.sha.0932da82f08175e906b0e2a8052120c823374e9f (64 Bit)
```

I have no idea why it messes like that.

I made a backup from the last stable code in a new branch (stable-pharo9). 

May I merge the changes?

**Pull request merged**